Lesson_26/task2.py

```diff
@@ -0,0 +1,116 @@
+"""Task 2 - Implement in (__contains__) and len (__len__) methods for HashTable"""
+
+
+class MyHashTable:
+    """Implementation of HastTable"""
+    def __init__(self, max_length=0):
+        self.max_length = max_length or 100
+        self.storage = []
+        for _ in range(self.max_length):
```

Comment:
Why not reducing it to `self.storage = [[]] * self.max_length` on the previous line? (that's a tricky question)